sample-operator/sample-operator.yaml

```diff
@@ -25,7 +25,6 @@ rules:
 - apiGroups:
```

Comment:
I'm wondering if this ClusterRole is a bit too permissive. for example, should it be able to read every secret in the cluster? delete nodes? create namespaces? etc. the etcd-operator specifies these permissions:

https://github.com/coreos/etcd-operator/blob/master/doc/user/rbac.md#create-clusterrole

Reply:
yeah, we could definitely scale down the permissions in the sample